L/LeastSquaresOptim/Compat.toml

```diff
@@ -16,11 +16,14 @@ ForwardDiff = "0.3-0.10"
 ["0.3-0.5"]
 julia = "0.6-0.7"
 
-["0.5-0.7"]
+["0.5-0.7.1"]
 Optim = "0.1-0.18"
 
-["0.6-0.7"]
+["0.6-0.7.1"]
 julia = "0.7-1.3"
 
-["0.6.1-0.7"]
+["0.6.1-0.7.1"]
 DiffEqDiffTools = "0.0-0.8"
+
+["0.7.2"]
+julia = "0.7-1.1"
```

Comment:
This is wrong, see https://github.com/JuliaComputing/Registrator.jl/issues/103. Linking it with that issue so we can correct it later.